src/GeneralizedTCRMapping.ts

```diff
@@ -0,0 +1,26 @@
+import { Address, BigInt, log, Bytes } from '@graphprotocol/graph-ts';
+import { ItemStatusChange, GeneralizedTCR } from '../generated/GeneralizedTCR/GeneralizedTCR';
+import { CuratedMarket } from '../generated/schema';
+
+
+export function handleItemStatusChange(event: ItemStatusChange): void {
+  const tcr = GeneralizedTCR.bind(event.address);
+  const itemInfo = tcr.getItemInfo(event.params._itemID);
+  const decodedData = itemInfo.value0.toString()
+
+  const addressStartIndex = decodedData.lastIndexOf('0x')
+  if (addressStartIndex == -1) return // Invalid submission. No Op
+  const fpmmAddress = decodedData.slice(decodedData.lastIndexOf('0x'))
```

Comment:
Could use `decodedData.slice(addressStartIndex, addressStartIndex + 42).toLowerCase()`.

Reply:
Good call. Unfortunately I couldn't just `String.toLowerCase` due to assemblyscript limitations, but its done.